t/16-utils-job-templates.t

```diff
@@ -28,9 +28,7 @@ my $template = {
 };
 my $errors = validate_data(%default_args, data => $template,);
 is scalar @$errors, 0, "Empty template - no errors";
-
-eval { my $errors = validate_data(schema_file => $invalid_schema, data => $template); };
-like($@, qr{JSON::Validator}, "Invalid schema file");
+throws_ok { my $errors = validate_data(schema_file => $invalid_schema, data => $template); } qr{JSON::Validator}, "Invalid schema file");
```

Comment:
Why is $errors set to the return value but then overwritten in the next statement as well?